main.go

```diff
@@ -474,3 +495,26 @@ func (nc *WriterCounter) Write(p []byte) (int, error) {
 	nc.bytes += int64(n)
 	return n, err
 }
+
+func getFnName() (string, error) {
+	name, ok := os.LookupEnv("OPENFAAS_NAME")
```

Comment:
Should `faas-cli local-run` inject this variable then? I don't think it does presently.

cc @rgee0